sections/05_treat.md

```diff
@@ -34,6 +34,16 @@ interpretability of deep learning models, fitting deep models to limited and
 heterogeneous data, and integrating complex predictive models into a dynamic
 clinical environment.
 
+A critical challenge in moving from prediction to treatment recommendations
+is the necessity to establish a causal relationship for a recommendation.
+Causal inference is often framed in terms of counterfactual question
+[@doi:10.1037/h0037350]. Johansson [@arxiv:1605.03661] use deep neural networks
+to create representation models for covariates that capture nonlinear effects
+and show significant performance improvements over existing models. In a less
+formal approach, Kale et al [@pmid:PMC4765623] first create a deep neural
```

Comment:
hmm - found the google scholar identifier and I also don't see a DOI on any of these versions:
https://scholar.google.com/scholar?oi=bibs&hl=en&cluster=4098913113177273200

Odd.